src/languages/sql.js

```diff
@@ -27,18 +27,24 @@ export default function(hljs) {
   const STRING = {
     className: 'string',
     variants: [
-    {
-      begin: '\'', end: '\'',
-      contains: [{begin: '\'\''}]
-    },
-    {
-      begin: '"', end: '"',
-      contains: [{begin: '""'}]
-    },
-    {
-      begin: '`', end: '`'
-    }
-  ]};
+      {
+        begin: /'/,
+        end: /'/,
+        contains: [
+          {begin: /''/ }
+        ]
+      },
+      {
+        begin: /`/,
+        end: /`/
+      }
+    ]
+  };
+  const QUOTED_IDENTIFIER = {
+    begin: /"/,
+    end: /"/,
+    contains: [ { begin: /""/ } ]
```

Comment:
@egor-rogov Any idea if this escaping is a thing for quoted identifiers or if it can be removed?

Reply:
The escaping is correct. But since identifiers are not to be highlighted, you can safely remove all this.

Reply:
_We need still to match to avoid false positives on keywords._

Reply:
Yes, you're right of course.